ion/parse_text_simple.go

```diff
@@ -56,15 +57,43 @@ func (p *parser) parseBinary(annotations []Symbol) Value {
 func (p *parser) parseLongString(annotations []Symbol) String {
 	text := []byte{}
 	for item := p.peekNonComment(); item.Type == lex.IonStringLong; item = p.peekNonComment() {
-		text = append(text, doStringReplacements(p.next().Val)...)
+		text = append(text, p.doStringReplacements(p.next().Val)...)
 	}
 
 	return String{annotations: annotations, text: text}
 }
 
+// decodeHex takes a slice that is a hex-encoded rune (up to 4 bytes large)
+// and decodes it into a slice containing UTF-8 code units.
+func (p *parser) decodeHex(hex []byte) []byte {
+	// must be a power of two and no larger that uint32
```

Comment:
Nitpick: `Must be a power of two and no larger than uint32.`

Reply:
Done.